src/share/streams_nuopc/dshr_strdata_mod.F90

```diff
@@ -870,48 +818,21 @@ subroutine shr_strdata_advance(sdat, ymd, tod, logunit, istr, timers, rc)
                      sdat%pstrm(ns)%ymdLB, sdat%pstrm(ns)%todLB, sdat%pstrm(ns)%ymdUB, sdat%pstrm(ns)%todUB
                 call shr_sys_abort(trim(subName)//' ERROR dt limit for stream')
              endif
-
-             ! If new data was read in, spatially interpolate the lower and upper bound data to the model grid
-             call t_startf(trim(lstr)//trim(timname)//'_map')
-             if (debug > 0) then
-                call dshr_fldbun_diagnose(sdat%pstrm(ns)%fldbun_stream_lb, subname//':fldbun_stream_lb',rc=rc)
-                if (ChkErr(rc,__LINE__,u_FILE_u)) return
-                call dshr_fldbun_diagnose(sdat%pstrm(ns)%fldbun_stream_ub, subname//':fldbun_stream_ub',rc=rc)
-                if (ChkErr(rc,__LINE__,u_FILE_u)) return
-             endif
-
-             ! regrid
-             call dshr_fldbun_regrid(sdat%pstrm(ns)%fldbun_stream_lb, sdat%pstrm(ns)%fldbun_model_lb, &
-                  sdat%pstrm(ns)%routehandle, rc=rc)
-             if (ChkErr(rc,__LINE__,u_FILE_u)) return
-             call dshr_fldbun_regrid(sdat%pstrm(ns)%fldbun_stream_ub, sdat%pstrm(ns)%fldbun_model_ub, &
-                  sdat%pstrm(ns)%routehandle, rc=rc)
-             if (ChkErr(rc,__LINE__,u_FILE_u)) return
-
-             if (debug > 0) then
-                call dshr_fldbun_diagnose(sdat%pstrm(ns)%fldbun_model_lb, subname//':fldbun_model_lb',rc=rc)
-                if (ChkErr(rc,__LINE__,u_FILE_u)) return
-                call dshr_fldbun_diagnose(sdat%pstrm(ns)%fldbun_model_ub, subname//':fldbun_model_ub',rc=rc)
-                if (ChkErr(rc,__LINE__,u_FILE_u)) return
-             endif
-             call t_stopf(trim(lstr)//trim(timname)//'_map')
           endif
 
           call t_stopf(trim(lstr)//trim(timname)//'_readLBUB')
 
        enddo ! end of loop over streams
-
+#ifdef DOTHIS
```

Comment:
So the vector field implementation has not been tested yet? If you have tested it - then we don't need this if-def. In fact it looks like you are handling the vector streams in another place - so why is this still here?